### PR TITLE
Sleep to snuze

### PR DESCRIPTION
**Pull request by sashman**
Changing sleep to snuze package. Using optional dependencies did not save npm from breaking.

### CODE REVIEW

**Comment by sashman** [comment]
`delay` in the `.usher.yml` has been changed to `delay_ms`

**Comment by SlowShip** [comment]
Nice :) How does Snuze work?

**Comment by sashman** [comment]
@SlowShip it's just a while loop

**Comment by SlowShip** [comment]
:+1:

**Comment by SlowShip** [comment]
Why the revert?

**Comment by sashman** [comment]
There are configs already using `delay` so it would have broken backwards compatibility.

File: src/modules/task-runner.js [hunk 1]
```diff
@@ -60,8 +51,8 @@ class TaskRunner {
       return true;
     }
 
-    if (this.shouldCommandRetry(command)) {
-      sleep.usleep(command.retry.delay * 1000000);
+    if(this.shouldCommandRetry(command)) {
+      snuze.snooze(command.retry.delay * 1000);
```
Comment (Steve-O-Cassels) [inline src/modules/task-runner.js]: Magic number - assumed microseconds - could this be a named variable e.g. snoozeMicroseconds?

**Comment by Steve-O-Cassels** [comment]
Minor readability - after that its a :+1:

**Comment by Angry-Potato** [comment]
:+1:

**Pull request merged**